src/lapack.jl

```diff
@@ -26,7 +26,7 @@ for (f, elty, relty) in (
             alpha::Real,
             A::StridedMatrix{$elty},
             beta::Real,
-            C::StridedVector{$elty},
+            C::StridedVecOrMat{$elty},
```

Comment:
```suggestion
            C::VecOrMat{$elty},
```
This is probably too wide a signature. The routine requires that all the memory is contiguous so I think this should be narrowed to VecOrMat